application/models/Permission.php

```diff
@@ -769,8 +769,7 @@ public function getUserId($iUserID = null)
                 /* Alt : return 1st forcedAdmin ? */
                 throw new Exception('Permission must not be tested with console application.');
             }
-            /* See TestBaseClass tearDownAfterClass */
```

Comment:
Maybe we need to add `@todo` to have its usage be replaced by App->getCurrentUserId ?

Reply:
@deprecated on the function ? Good idea !

Reply:
Not this one, this one is linked to Permission : Permission must not test console application currently.

Or only on a 7.X version

Reply:
Not sure why ... but I will follow you.. :) Don't add comment then :)